test/Azure.Functions.Cli.Tests/E2E/TestProject/TestNet6InProcProject/.gitignore

```diff
@@ -0,0 +1,262 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Don't we have a gitignore in root level? Is that not enough?

Reply:
removed gitignore